content/kubernetes/re-databases/routes.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: Use OpenShift routes for external database access
+linkTitle: OpenShift routes
+description:  
+weight: 15
+alwaysOpen: false
+categories: ["Platforms"]
+aliases: [
+   /kubernetes/re-databases/routes.md,
+   /kubernetes/re-databases/routes/,
+]
+---
+
+Every time a Redis Enterprise database (REDB) is created with the Redis Enterprise operator, a [service](https://kubernetes.io/docs/concepts/services-networking/service/) is created that allows requests to be routed to that database. Redis Enterprise supports three [types of services](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) for accessing databases: `ClusterIP`, `headless`, or `LoadBalancer`.
```

Comment:
Does the operator create the database? Does the operator also create the service? If so, maybe reword this slightly?
```suggestion
Every time the Redis Enterprise operator creates a Redis Enterprise database (REDB), it creates a [service](https://kubernetes.io/docs/concepts/services-networking/service/) that allows requests to be routed to that database. Redis Enterprise supports three [types of services](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) for accessing databases: `ClusterIP`, `headless`, or `LoadBalancer`.
```